spec/inputs/jdbc_spec.rb

```diff
@@ -410,6 +412,7 @@
         event = queue.pop
         expect(event.get("num")).to eq(12)
         expect(event.get("custom_time").time).to eq(Time.iso8601("2015-01-02T03:21:21.000Z"))
+        Timecop.return
```

Comment:
When I added this test I took care to use the block form of `Timecop.travel` because the docs said it does the return after the block yield.

Did you verify that this particular `Timecop.return` was necessary? If so, we should raise an issue in the Timecop library.

Reply:
This one was not the issue... i just did a quick search for Timecop without a return. I will revert this line.